src/Jackett.Common/Definitions/uhdbits.yml

```diff
@@ -133,10 +133,16 @@ search:
         - name: replace
           args: [" / 2x", ""]
         - name: re_replace
-          args: [" / ", " "]
+          args: [" / ", "-"]
         #  #10423
```

Comment:
I don't agree with replacing ` / ` with `-`
`Cyberpunk: Edgerunners S01 [2022] 1080p-​HEVC-​WEB-​DL-​DoVi-​HDR10-​EFx`
is not easier to parse than
`Cyberpunk: Edgerunners S01 [2022] 1080p HEVC WEB-​DL DoVi HDR10 EFx`
IMO I doubt sonarr or radarr will parse this any differently and for users of the dashboard search its looks worse than having spaces

Reply:
Fair point, perhaps there's a way to improve the regex to only apply the dash to the final space.

However - Sonarr and Radarr are not able to recognize a group name without the group being preceded by a "-".  This problem is compounded when using Trash's guides as UHDits results are never properly interpreted without the group name.

Reply:
so ideally you would like 
`The Crazies [2010] 1080p-​x264-​Blu-​Ray-​TayTO`
 turned into
`The Crazies [2010] 1080p x264 Blu-​Ray -​TayTO`
?

Reply:
Yes, that would be a preferable solution.